src/Tribe/Editor/Provider.php

```diff
@@ -65,6 +65,12 @@ protected function hook() {
 			return false;
 		}
 
+		$hooks = new \Tribe\Events\Editor\Hooks;
+		$hooks->register();
```

Comment:
Avoid this, turn the Hooks class into a provider.
```suggestion
		$this->container->register( \Tribe\Events\Editor\Hooks::class )
```